CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- The [Swagger extension](reference/OpenAPI/differences.md#x-flat-extensions) `x-flat-validate` is now also recognized below `paths/<path>` and `paths/<path>/<operation>`.
```

Comment:
Hmm, in github muss die Referenz nach `#x-flat--extensions` gehen, in gitbook nach `#x-flat-extensions`. Ob gitbook das automatisch passend macht?

Reply:
Kannst Du ausprobieren. Ich hatte in Kauf genommen, dass der Link in Github nicht funktioniert.